resume.json

```diff
@@ -0,0 +1,189 @@
+
+,"highlights":["Specify multiple accomplishments"]}]
+ ,"education":[{
+"institution":"XYZ Institute of Technology - [Add institute name]"
+
+,"area":"Engineering"
+
+,"studyType":"Bachelor"
+
+,"startDate":"2017-06-29"
+
+,"endDate":"2013-06-29"
+
+,"score":"eg. GPA or PERCENTAGE - [Add score type]"
+
+,"courses":["List notable courses/subjects"],"honors":["List education honours"]}]
+ ,"volunteer":[{
+"organization":"Xyz "
+
+,"position":"Open Source Contributor [Contribution type]"
+
+,"location":{
+ "lat": "0",
+"long": "0"
+}
+
+,"url":"http://xyz.example.com"
+
+,"startDate":"2014-06-29"
+
+,"endDate":"2017-06-29"
+
+,"summary":"Give an overview of your responsibilities at the company"
+
+,"highlights":["Specify accomplishments and achievements"]}]
+ ,"publications":[{
+"name":"Deep learning and Artificial Intelligence"
+
+,"publisher":"XYZ, Computer Magazine"
+
+,"releaseDate":"2015-08-01 - [resume.json uses the ISO 8601 date standard]"
+
+,"resources":["Specify multiple resources with label"],"url":"http://www.computer.org.example.com/csdl/mags/co/2015/10/rx069-abs.html"
+
+,"summary":"Discussion of the advent of deep learning and artificial intelligence - Short summary of publication"
+
+}]
+ ,"legal":[{
+"name":"XYZ's patent on LZW compression, a fundamental part of the widely used GIF graphics format - [Add document name]"
+
+,"legalType":"Patent, Trademark, Copyright - Give the type of this document"
+
+,"description":"Give a brief description about this document"
+
+,"applicationDate":"2015-08-01"
+
+,"grantDate":"2016-09-01"
+
+,"endDate":"2020-09-03"
+
+,"resources":["Specify multiple resources with label"],"idNumber":"JP2004369746A - [Add the application number or Id Number]  "
+
+}]
+ ,"skills":[{
+"name":"Web Development"
+
+,"keyword":["List some keywords pertaining to the skill"]}]
+ ,"awards":[{
+"title":"Awarded Software Process Achievement Award "
+
+,"date":"2016-06-12"
+
+,"awarder":" IEEE"
+
+,"summary":"Received for my work in Deep learning and AI"
+
+}]
+ ,"projects":[{
+"name":"File Transfer application - [Name of the project]"
+
+,"location":{
+ "lat": "0",
+"long": "0"
+}
+
+,"description":"Developed a client and server based application - [Short summary of project]"
+
+,"highlights":["Specify multiple features"],"keywords":["Specify special elements involved"],"startDate":"2017-06-29"
+
+,"endDate":"2017-06-29"
+
+,"resources":["Specify multiple resources with label"],"url":""
+
+,"roles":["Specify your role on this project or in company"],"entity":"'greenpeace', 'corporationXYZ' - [Relevant company/entity affiliations]"
+
+,"type":"'volunteering', 'presentation', 'talk', 'application', 'conference'"
+
+}]
+ ,"certificates":[{
+"code":"1Z0-062"
+
+,"name":"XYZ Certified Application Specialist (MCAS) - [Add the certificate name]"
+
+,"website":""
+
+,"verification":""
+
+,"grantDate":"2017-06-29"
+
+,"score":"Exam result (PASS/FAIL, 100%, 100)"
```

Comment:
Not a "valid" value.

Reply:
@stp-ip  what value can we actually use here for example?
https://github.com/resumic/schema/blob/master/schema.json#L546

Reply:
I would say let's start with a percentage. It's the most portable for now I guess.